internal/services/connections/client/client.go

```diff
@@ -14,15 +16,21 @@ type Client struct {
 	ManagedApisClient *managedapis.ManagedAPIsClient
 }
 
-func NewClient(o *common.ClientOptions) *Client {
-	connectionsClient := connections.NewConnectionsClientWithBaseURI(o.ResourceManagerEndpoint)
-	o.ConfigureClient(&connectionsClient.Client, o.ResourceManagerAuthorizer)
+func NewClient(o *common.ClientOptions) (*Client, error) {
+	connectionsClient, err := connections.NewConnectionsClientWithBaseURI(o.Environment.ResourceManager)
+	if err != nil {
+		return nil, fmt.Errorf("build connections Client: %+v", err)
```

Comment:
Can we be consistent with the error messages and casing
```suggestion
		return nil, fmt.Errorf("building Connections client: %+v", err)
```